ui/src/bin/copy_data.js

```diff
@@ -67,21 +67,32 @@ const run = async () => {
 	);
 
 	const types = await readFile(TYPES_PATH).then(yaml.safeLoad);
-	const tmpAllIndicatorsCsvPath = tempy.file({name: allIndicatorsCsvName});
+	const tmpAllIndicatorsCsvPath = tempy.file({
+		name: `${basename}_${regionType}.csv`
+	});
 
 	await Promise.all(
-		specs.map(spec =>
-			readCsv(spec.csvFilepath, transformValues({
-				[spec.order[0]]: Number,
-				[spec.order[2]]: Number,
+		specs.map(spec => {
+			const [yearHeader, regionIdHeader, regionYearSpecHeader,] = spec.order;
+
+			return readCsv(spec.csvFilepath, transformValues({
+				[yearHeader]: Number,
+				[regionYearSpecHeader]: Number,
 				[spec.schema.value.id]: Number,
 			}))
 			.then(_.mapWith(applyFnMap({
 				id: () => spec.schema.value.id,
-				year: _.getKey('year'),
-				nuts_id: _.getKey('nuts_id'),
-				nuts_year_spec: _.getKey('nuts_year_spec'),
-				nuts_name: obj => NUTS2_UK_labels[obj.nuts_year_spec][obj.nuts_id],
+				[yearHeader]: _.getKey(yearHeader),
+				[regionIdHeader]: _.getKey(regionIdHeader),
+				[regionYearSpecHeader]: _.getKey(regionYearSpecHeader),
+				name: obj => {
```

Comment:
Minor: Is it possible to dynamically name this column such that it gives `lep_name` or `nuts_name`?

Reply:
Or maybe just `region_name` for all